Composer/plugins/localPublish/src/index.ts

```diff
@@ -317,13 +388,15 @@ class LocalPublisher implements PublishPlugin<PublishConfig> {
 
     child.on('exit', (code) => {
       if (code !== 0) {
-        reject(erroutput);
+        // this.setBotStatus(botId, { status: 500, result: { message: erroutput } });
```

Comment:
Are there scenarios where the non-zero code does not indicate some sort of error?

How can we properly handle this? I don't like just silently eating this

Reply:
add this back,  and remove event listener before killing process to solve the non-zero exit code.